src/utils/formatSearchResponse.ts

```diff
@@ -136,7 +136,6 @@ const getOtherDetails = async (
     metadata_standard: searchItem?._source?.standardNameObject?.default ?? '',
     project_number: projectId ?? '',
     metadata_language: searchItem?._source?.mainLanguage ?? '',
-    ncea_catalogue_date: formatDate(searchItem?._source?.dateStamp, false, false, '-'),
```

Comment:
Please ensure that this is removed from anywhere it has been used.

Reply:
Done